### DISCUSSION TITLE
Help me understand the "Left to spend" box

### DISCUSSION THREAD

**Discussion by M4RC02U1F4A4** [firefly-iii/firefly-iii#5170] [Firefly III] [answered]
Hi, I've read other posts but can't find an answer to my question

This is what i have understand
- I create one budget for 100€/month
![immagine](https://user-images.githubusercontent.com/23193188/136715031-a8e40140-517b-452b-8e84-487683741300.png)
- Now i make a withdrawal for 10€ and add it to the budget
![immagine](https://user-images.githubusercontent.com/23193188/136715038-c241b8ad-aa2d-4c09-b4ee-a728c0389d8d.png)
![immagine](https://user-images.githubusercontent.com/23193188/136715048-96b0c98e-7a54-4288-9eb7-c4e7fc305423.png)

What i expect is to see the "Left to spend" box at 90€, but i see -10€, why this?
![immagine](https://user-images.githubusercontent.com/23193188/136715054-58250a13-bda4-4fac-80f0-d98147a6f347.png)

**Comment by JC5** [comment, marked as answer]
If you set the "total available amount" on the `/budgets` tab, you'll see the "left to spend" correct itself 👍🏻

**Reply by M4RC02U1F4A4** [reply]
I Got It. Thanks!
But now I am wondering if there is a way to increase the "total available amount" automatically.
If I create a budget that increases by € 100 per month, do I have to manually increase the "total available amount" each month? I would expect my monthly budget to be the sum of all budgets

**Reply by JC5** [reply]
In Firefly III it works the other way around. You could have 1000 available, so start with that and divide it over your budgets. The bars will change accordingly.
